src/js/login.jsx

```diff
@@ -16,96 +17,88 @@ const PageContainer = styled.div`
+            type={"text"}
+            value={username}
+          />
 
-  render() {
-    const {
-      localeText: { users },
-    } = this.context;
-    return (
-      <ThemeProvider theme={THEME}>
-        <PageContainer>
-          <AccountForm header={users?.loginTitle} submitFn={this.requestLogin}>
-            {this.renderField(users?.username, "text", "username")}
-            {this.renderField(users?.password, "password", "password")}
-            <div style={{ display: "flex", justifyContent: "space-between", alignItems: "center" }}>
-              <a href="/password-request">{users?.forgot}</a>
-              <Button extraStyle={{ marginTop: "1rem" }}>{users?.login}</Button>
-            </div>
-            <div style={{ display: "flex", flexDirection: "column", alignItems: "center" }}>
-              <h3>{users?.newUser}</h3>
+          <TextInput
+            id="password"
+            label={t("users.username")}
+            onChange={(e) => setPassword(e.target.value)}
+            placeholder={`Enter ${(t("users.username") || "").toLowerCase()}`}
+            type="password"
+            value={password}
+          />
+          <div style={{ display: "flex", justifyContent: "space-between", alignItems: "center" }}>
+            <a href="/password-request">{t("users.forgot")}</a>
+            <Button extraStyle={{ marginTop: "1rem" }}>{t("users.login")}</Button>
+          </div>
+          <div style={{ display: "flex", flexDirection: "column", alignItems: "center" }}>
+            <h3>{t("users.newUser")}</h3>
+            <div>
               <div>
-                <div>
-                  <Button
-                    onClick={(e) => {
-                      e.preventDefault();
-                      window.location = "/register";
-                    }}
-                  >
-                    {users?.register}
-                  </Button>
-                </div>
+                <Button
+                  onClick={(e) => {
+                    e.preventDefault();
+                    window.location = "/register";
+                  }}
+                >
+                  {t("users.register")}
+                </Button>
               </div>
             </div>
-          </AccountForm>
-        </PageContainer>
-      </ThemeProvider>
-    );
-  }
+          </div>
+        </AccountForm>
+      </PageContainer>
+    </ThemeProvider>
+  );
 }
 
-Login.contextType = MainContext;
-
 export function pageInit(args) {
   ReactDOM.render(
-    <PageLayout
-      role={args.role}
-      userLang={args.userLang}
-      username={args.username}
-      version={args.versionDeployed}
-      showSearch={false}
-    >
-      <Login />
-    </PageLayout>,
+    <Suspense fallback="loading">
+      <PageLayout
+        role={args.role}
+        username={args.username}
+        mapboxToken={args.mapboxToken}
+        mapquestKey={args.mapquestKey}
+        versionDeployed={args.versionDeployed}
+        showSearch={true}
```

Comment:
This should be `showSearch={false}`. I also don't think we need to provide the `mapboxToken` or `mapquestKey` here